tools/seed/main.go

```diff
@@ -587,3 +607,29 @@ func createFirebaseUser(ctx context.Context, firebaseAuth *firebaseauth.Client,
 
 	return nil
 }
+
+// createSecrets creates secrets. It re-uses the rotation worker logic and
+// invokes the handler.
+func createSecrets(ctx context.Context, db *database.Database, keyManager keys.SigningKeyManager, secretManager secrets.SecretVersionManager) error {
+	cfg, err := config.NewRotationConfig(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to create rotation config: %w", err)
+	}
+
+	h, err := render.New(ctx, nil, cfg.DevMode)
+	if err != nil {
+		return fmt.Errorf("failed to create renderer: %w", err)
+	}
+
+	rotationController := rotation.New(cfg, db, keyManager, secretManager, h)
+
+	if err := rotationController.RotateTokenSigningKey(ctx); err != nil {
+		return fmt.Errorf("failed to create initial token signing key: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to rotate token signing key: %w", err)
```
are these always initial?

Reply:
I don't think it's actually supported, but _technically_ if you ran the seed script again, it would rotate them.